app/src/main/java/com/chouten/app/presentation/ui/ChoutenAppViewModel.kt

```diff
@@ -2,42 +2,56 @@ package com.chouten.app.presentation.ui
 
 import android.content.Context
 import android.net.Uri
-import android.os.Build
-import androidx.core.net.toUri
 import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
 import com.chouten.app.domain.model.SnackbarModel
-import com.chouten.app.domain.proto.FilePreferences
 import com.chouten.app.domain.proto.filepathDatastore
-import com.chouten.app.domain.repository.SnackbarRepository
+import com.chouten.app.domain.use_case.module_use_cases.ModuleUseCases
```

Comment:
this is some odd formatting, shouldn't it be `com.chouten.app.domain.useCase.moduleUseCases.ModuleUseCases`?

Reply:
(same goes for the folders behind this ofc)

Reply:
Yes, we need to stick to only one formatting

Reply:
https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html
Following the advice here, the current way of doing things seems appropriate. Package names are snake cased, and other files/classes pascal case.